test/config_init.json.dist

```diff
@@ -1,6 +1,6 @@
 {
-  "originWeb3Provider": "ws://localhost:8547",
-  "auxiliaryWeb3Provider": "ws://localhost:8548",
+  "originWeb3Provider": "ws://167.99.252.47:8646",
+  "auxiliaryWeb3Provider": "ws://167.99.252.47:8647",
```

Comment:
These ports shouldn't be open anymore. We should probably not distribute this IP and ports.

Reply:
@schemar What IP and ports should be used?